types/accedo__accedo-one/index.d.ts

```diff
@@ -0,0 +1,409 @@
+// Type definitions for @accedo/accedo-one 4.0
+// Project: https://www.accedo.tv/one
+// Definitions by: Alexander P. Cerutti <https://github.com/alexandercerutti>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
+
+// tslint:disable-next-line no-single-declare-module
```

Comment:
I don't see a reason to do this.

Reply:
@RyanCavanaugh As I wrote in the notes, I had to insert it because `dtslint` was throwing me the error for `declare module`. I've inserted it in the tslint.json, but then I read the checklist here (6th check) on Github and then I moved it on the row.

Also, again, as I said, this library supports UMD (which, as I understood, does not use `declare module`, but `export as namespace` and `export =`). But if I try to use them, VSCode is not able to find the declarations in the project I'm working on.

I don't know how to solve this.

Reply:
@RyanCavanaugh I had to insert it because `dtslint` was throwing me the error for declare module. I inserted it in the tslint.json, but then I read the checklist here (6th check) on Github and then I moved it on the row.

Also, as I wrote in the notes, this library supports UMD (which, as I understood, does not use declare module, but export as namespace and export =). But if I try to use them, VSCode is not able to find the declarations in the project I'm working on.

I don't know how to solve this.

Reply:
@RyanCavanaugh I've removed that line as you asked, but obliviously now the tests fail because the linting fails. How can I fix this? Thank you.